index.html

```diff
@@ -97,7 +100,7 @@ <h1>GAMERS</h1>
     </nav>
     <header class="pb-4">
       <div class="container">
-        <div class="row">
+        <div class="row ">
           <div class="col-md-6 d-flex flex-column text-contorller">
             <h2>PS5 V2 <br />DualShock 5</h2>
             <p>
```

Comment:
"Latest Products" => Space between tabs and cards are not like the design

Reply:
Done